contracts/examples/crypto-kitties/kitty-auction/src/lib.rs

```diff
@@ -71,16 +71,6 @@ pub trait KittyAuction {
 		}
 	}
 
-	#[endpoint]
-	fn claim(&self) -> SCResult<()> {
```

Comment:
I asked him to do it :) The endpoint isn't needed at all for that contract, and it could lead to owner "stealing" auction locked funds.

Copy-paste oversight on my part when writing those contracts.

Reply:
Dorin kindly asked me. The owner is not supposed to be able to do this.